packages/fluentui/docs/src/components/ComponentDoc/ComponentPerfExample/ComponentPerfExample.tsx

```diff
@@ -14,16 +15,45 @@ export interface ComponentPerfExampleProps {
 
 const ComponentPerfExample: React.FC<ComponentPerfExampleProps> = props => {
   const { title, description, examplePath } = props;
-  // FIXME: find a better way
+
+  const [currentChart, setCurrentChart] = React.useState<'perf' | 'resources'>('perf');
+  const onShowPerfChart = React.useCallback(() => {
+    setCurrentChart('perf');
+  }, []);
+  const onShowResourcesChart = React.useCallback(() => {
+    setCurrentChart('resources');
+  }, []);
```

Comment:
```suggestion
  const [currentChart, setCurrentChart] = React.useState<'perf' | 'resources'>('perf');

  const onShowPerfChart = () => setCurrentChart('perf');
  const onShowResourcesChart = () => setCurrentChart('resources');
```

I don't see there any need to have `useCallback` here...